index.js

```diff
@@ -55,6 +55,9 @@ function ratelimit(opts) {
 
     var delta = (limit.reset * 1000) - Date.now() | 0;
     var after = limit.reset - (Date.now() / 1000) | 0;
+    if (opts.throwable) {
+      throw opts.throwable(after,ms(delta, { long: true }));
```

Comment:
it is better to throw an error like:

```
this.throw(429, 'Rate limit exceeded, retry in ' + ms(delta, { long: true }), {
  after: after,
  delta: delta,
  code: RATE_LIMIT
})
```

and you can catch this error outside.